server/chat-plugins/chatlog.ts

```diff
@@ -1333,4 +1333,29 @@ export const commands: ChatCommands = {
 		const [type, userid] = target.split(',').map(toID);
 		return this.parse(`/j view-logsaccess-${type || 'all'}${userid ? `-${userid}` : ''}`);
 	},
+	gcsearch: 'groupchatsearch',
+	async groupchatsearch(target, room, user) {
+		this.checkCan('lock');
+		// eslint-disable-next-line no-useless-escape
```

Comment:
?

Reply:
For some reason, ESLint sees the \- as a useless escape, and so fails checks on it.

Reply:
You don't need to escape `-` in character classes unless it's between, a and z, A and Z, or 0 and 9.

Reply:
That's because it _is_ a useless escape.